src/components/ContentPreview/ContentPreview.js

```diff
@@ -73,7 +73,8 @@ type Props = {
     onMetric?: Function,
     requestInterceptor?: Function,
     responseInterceptor?: Function,
-    previewInstance?: any
+    previewInstance?: any,
+    getUserProfileUrl?: Function
```

Comment:
Could you create a more descriptive type for this? 

IE) that it returns a promise that resolves in a string?